airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java

```diff
@@ -31,6 +33,14 @@
 
     static final String FAILURE_URL = "/login?error=1";
 
+    private static final String key;
+
+    static {
+      byte[] array = new byte[32];
+      new SecureRandom().nextBytes(array);
+      key = new String(array);
+    }
+
```

Comment:
This isn't bad, but maybe a cleaner way to do this is to create a `public static String generateRememberMeKey() {...}` function that you then call once inline like `.and().rememberMe().key(generateRememberMeKey())`. This would allow us to move this logic around easier if needed.

Reply:
The issue is that `generateRememberMeKey()` would return a different value each time, but I guess that since the `configure` methods are only called once, it should be ok.